website/pages/docs/concepts/seal.mdx

```diff
@@ -84,80 +84,129 @@ to decrypt the master key at startup and unseal automatically.
 When using Auto Unseal there are certain operations in Vault that still
 require a quorum of users to perform an operation such as generating a root token.
 During the initialization process, a set of Shamir keys are generated that are called
-Recovery Keys and are used for these operations.
+recovery keys and are used for these operations.
 
 For a list of examples and supported providers, please see the
 [seal documentation](/docs/configuration/seal).
 
+## Recovery Key Rekeying
+
+During the KMS Seal initialization process, a set of Shamir keys called recovery keys are
+generated which are used for operations that still require a quorum of users.
+
+Recovery keys can be rekeyed to change the number of shares or thresholds. When using the
+Vault CLI, this is performed by using the `-target=recovery` flag to `vault operator rekey`.
+
 ## Seal Migration
 
-The seal can be migrated from Shamir keys to Auto Unseal, Auto Unseal to Shamir
-keys and Auto Unseal to another Auto Unseal.
+The seal can be migrated from Shamir Seal to KMS Seal, KMS Seal to Shamir Seal,
+and KMS Seal to another KMS Seal.
+
+~> **NOTE**: Seal migration process cannot be performed without downtime. Due to
+the technical underpinnings of the seal implementations, it is at this point not
+possible to perform seal migration without briefly bringing the whole cluster
+down. We understand that it can be hard for many deployments to face downtime,
+but we believe that switching seals is a rare event and hence we hope for the
+downtime to be considered as an acceptable trade-off.
+
+~> **NOTE**: Seal migration operation will require both old and new seals to be
+available during the migration. For example, migration from KMS seal to Shamir
+seal will require that the KMS key be accessible during the migration.
+
+~> **NOTE**: Seal migration from KMS seal to Shamir seal is not currently
+supported when using Vault Enterprise. We plan to support this officially in a
+future release.
+
+~> **NOTE**: Seal migration from KMS seal to KMS seal of same kind is not
+currently supported when using Vault Enterprise. We plan to support this
```

Comment:
I think this is unsupported in OSS too, right?

Reply:
Correct. Updated.